lib/actions/help_ts.ts

```diff
@@ -24,8 +24,9 @@ export function getOclifHelpLinePairs(): Array<[string, string]> {
 	// improvement would probably be to automatically scan the actions-oclif
 	// folder.
 	const EnvAddCmd = require('../actions-oclif/env/add').default;
+	const EnvRmCmd = require('../actions-oclif/env/rm').default;
 	const VersionCmd = require('../actions-oclif/version').default;
-	return [EnvAddCmd, VersionCmd].map(getCmdUsageDescriptionLinePair);
+	return [EnvAddCmd, EnvRmCmd, VersionCmd].map(getCmdUsageDescriptionLinePair);
```

Comment:
Just FYI, an improvement I thought about in the past (not necessary to implement in this PR, unless you felt like doing it...) is to "centralise" the changes needed to add an oclif command in a single place. I mean, as you have noticed, you've had to add/modify lines in `help_ts.ts`, `app.ts`, `oclif-utils.ts` and `capitanodoc.ts`, so it feels a bit like "all over the place". I wonder if it could be all in an array or object in, say, `oclif-utils.ts`, then imported by the other files. But it's just a thought.

Reply:
@pdcastro yeah I noticed this, and I have some ideas of how we can improve the process. I will probably send a follow up PR soon :smile:

Reply:
see https://github.com/balena-io/balena-cli/issues/1391